lib/paperclip.rb

```diff
@@ -69,7 +69,7 @@
 require 'logger'
 require 'cocaine'
 
-require 'paperclip/railtie' if defined?(Rails)
+require 'paperclip/railtie' if defined?(Rails::Railtie)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.